### PR TITLE
New IEx (Interactive Elixir) Cheat Sheet.

### PR DESCRIPTION
**Pull request by pjhampton**
## IEx Cheat Sheet

This is the IEx (Interactive Elixir) cheat sheet for various shell helpers. 

https://duck.co/ia/view/iex_cheat_sheet

### Description
Helpers cheat sheet for Elixir's interactive shell (IEx).

### Data Source
http://elixir-lang.org/docs/stable/iex/IEx.Helpers.html

### Example Queries
iex cheat sheet

**Screenshot 1 (Default Theme) - Not Collapsed**
screen shot 2016-02-23 at 21 14 57

**Screenshot 2 (Dark Theme) - Collapsed**
screen shot 2016-02-23 at 21 15 32

https://duck.co/ia/view/iex_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@pjhampton Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

File: share/goodie/cheat_sheets/json/iex.json [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    "aliases": [
+        "iex",
+        "interactive elixir"
+    ],
+    "template_type": "code",
+    "section_order": [
+        "IEx Commands"
+    ],
+    "sections": {
+        "IEx Commands": [
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/iex.json]: If possible it'd be best to split this into 2+ categories to improve the layout

Comment (pjhampton) [inline share/goodie/cheat_sheets/json/iex.json]: @moollaza on it!

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=iex cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/iex_cheat_sheet_mobile.png)

**Comment by GuiltyDolphin** [comment]
@moollaza @pjhampton Whoops! We need to remove the `iex` alias - otherwise we're gonna get some failing tests!

**Comment by pjhampton** [comment]
@GuiltyDolphin How come Ben?

**Comment by GuiltyDolphin** [comment]
@pjhampton @moollaza Edit: @moollaza just PR'd a fix.

Duplicating the cheat sheet name in the aliases is a failure as of recently. It didn't get caught on this because the tests weren't re-run.

**Comment by moollaza** [comment]
Yes sorry guys! I jumped the gun and merged this PR when I meant to do the other Elixr Cheat Sheet, then I saw the failing test and fixed

**Pull request merged**